### PR TITLE
Remove unnecessary step to manually update the service account secrets

### PR DESCRIPTION
**Pull request by rohitagarwal003**
Playing with v1.19.16, it seems that updating `--root-ca-file` flag in the kube-controller-manager config and then restarting it results in all those secrets getting updated with the new value.

I will let someone from SIG-Auth verify my empirical evidence.

/cc @enj @mikedanese @liggitt

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
@kubernetes/sig-auth-pr-reviews : Hello, please can y'all take a look at this?

**Comment by zacharysarah** [comment]
@kubernetes/sig-auth-pr-reviews Ping

**Comment by divya-mohan0209** [comment]
@kubernetes/sig-auth-pr-reviews : Please could you review this?

**Comment by kbhawkey** [comment]
Hello @rohitagarwal003 . Would you rebase your changes? Thanks.

**Comment by divya-mohan0209** [comment]
Hello @rohitagarwal003 , please can you update whether you'd be able to rebase & incorporate the changes?

**Comment by rohitagarwal003** [comment]
Hello,

Thanks for the reviews. I don't think I will have cycles to rebase and fix
the conflicts for a couple of weeks. But I have checked the "Allow edits by
maintainers" box on the PR, so you should be able to add commits to this
branch to fix the conflicts if you have cycles. If not, I can come back and
address the conflicts later.

--
Rohit

**Comment by divya-mohan0209** [comment]
Thank you @rohitagarwal003 for your response! I completely understand and we'll be closing off this PR for now. Whenever you have the time, please feel free to reopen the PR and work on it :) 
/close

File: content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md [hunk 1]
```diff
@@ -46,22 +46,10 @@ Configurations with a single API server will experience unavailability while the
    a bundle you can restore the problem flags to point to `ca.crt` and delete the copy.
    {{< /note >}}
 
-1. Update all service account tokens to include both old and new CA certificates.
+1. Wait for controller-manager to update `ca.crt` in the service account secrets to include both old and new CA certificates.
```
[the remark this answers is not in the document]
Comment (rohitagarwal003) [reply content/en/docs/tasks/tls/manual-rotation-of-ca-certificates.md]: Done

**Comment by rohitagarwal003** [comment]
This is ready to be merged.

**Comment by rohitagarwal003** [comment]
@divya-mohan0209 Ping

**Comment by rohitagarwal003** [comment]
Hello, can someone merge this before I get merge conflicts again?

@kubernetes/sig-docs-en-reviews

**Comment by shannonxtreme** [comment]
(I can't approve or merge though, a PR wrangler will need to. This week's PR wrangler is Brad)

/assign @bradtopol

**Pull request merged**